UnitySetup/UnitySetup.psm1

```diff
@@ -1870,7 +1870,23 @@ function Start-UnityEditor {
             $process.Start() | Out-Null
 
             if ( $Wait ) {
-                $process.WaitForExit()
+                if( $LogFile ) {
+                    New-Item -Path $LogFile -ItemType File -Force
+                    $ljob = Start-Job -ScriptBlock { param($log) Get-Content "$log" -Wait } -ArgumentList $LogFile
+
+                    while ( -not $process.HasExited -and $ljob.HasMoreData )
+                    {
+                        Receive-Job $ljob
+                        Start-Sleep -Milliseconds 100
```

Comment:
```suggestion
```

not waiting doesn't seem to affect perf. Gonna remove it to get updates as they come